action.yml

```diff
@@ -205,6 +252,39 @@ runs:
       path: .tmp-ansible-collection-checkout
       persist-credentials: false
       ref: ${{ inputs.git-checkout-ref }}
+      fetch-depth: >-
+        ${{
+          steps.compute-change-detection-coverage.outputs.change-detection-arg
+          && '0' || '1'
+        }}
+
+  - name: Log the next step action
+    if: >-
```

Comment:
Maybe also check if the event is a pull request?

Reply:
`steps.compute-change-detection-coverage.outputs.change-detection-arg` is empty (and thus falsly) when the event is not a PR.